cluster/node-pools/master-default/userdata.yaml

```diff
@@ -455,13 +441,13 @@ write_files:
             h: Path("/kube-system/healthz")
               -> setPath("/healthz")
               -> disableAccessLog()
-              -> "http://127.0.0.1:8080";
+              -> "https://127.0.0.1:443";
```

Comment:
We should be able to remove these then because they were only to bypass authentication. (See the equal `all` rule below.)

Reply:
Also you can replace the custom RBAC rules with this: https://github.com/zalando-incubator/kubernetes-on-aws/pull/4007/commits/b15ada51e28b2a634b9a1b41d79b9035880303c6#diff-829c7c7be2a35683dae916778f24edfe9e91ff89fd17d07bc18f25953a0c49b8

There's a predefined rule to allow OIDC discovery access.

Reply:
Also you can replace the custom RBAC rules with this: https://github.com/zalando-incubator/kubernetes-on-aws/pull/4007/commits/b15ada51e28b2a634b9a1b41d79b9035880303c6#diff-829c7c7be2a35683dae916778f24edfe9e91ff89fd17d07bc18f25953a0c49b8

There's a predefined role to allow OIDC discovery access.

Reply:
Thanks for pointing it out! https://github.com/zalando-incubator/kubernetes-on-aws/pull/4145